lib/compat/wordpress-6.8/template-parts.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
The file name is a bit weird but it's not important. Could be rest-api.php or something.

Reply:
Done! b2c0bf587e30c302d6dc0bda91881933b87e3984